src/util/git/git-spawn.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+
+import * as child from '../child.js';
+
+// Suppress any password prompts since we run these in the background
+const env = {
+  GIT_ASKPASS: '',
+  GIT_TERMINAL_PROMPT: 0,
+  GIT_SSH_COMMAND: 'ssh -oBatchMode=yes',
+  ...process.env,
+};
+
+export const spawn = (args: Array<string>, opts?: child_process$spawnOpts = {}): Promise<string> => {
```

Comment:
I agree with @BYK I think it will be better if we move everything in the git module. I think you can check http://facebook.github.io/jest/docs/mock-function-api.html#content for some examples (or [here](https://github.com/facebook/jest/issues/936) if you have some doubts, they discuss a lot on this topic).
Btw great work! :D